pkg/oc/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -303,7 +324,7 @@ func (o *ReconcileClusterRoleBindingsOptions) ReplaceChangedRoleBindings(changed
 	return kutilerrors.NewAggregate(errs)
 }
 
-func computeUpdatedBinding(expected authorizationapi.ClusterRoleBinding, actual authorizationapi.ClusterRoleBinding, excludeSubjects []kapi.ObjectReference, union bool) (*authorizationapi.ClusterRoleBinding, bool) {
+func computeUpdatedBinding(expected rbac.ClusterRoleBinding, actual rbac.ClusterRoleBinding, excludeSubjects []rbac.Subject, union bool) (*rbac.ClusterRoleBinding, bool) {
```

Comment:
Why does this make shallow copies?

Reply:
Why are shallow copies a problem and how do I avoid a shallow copy?